doc/en/developer/source/release/guide/index.rst

```diff
@@ -102,7 +102,7 @@ When creating the first release candidate of a series, there are some extra step
     
     ant -f build/rename.xml 
     
-  .. note:: use of sed
+  .. note:: Alternative use of sed
```

Comment:
Should this whole note not simply be removed, or do some people still prefer sed over build/rename.xml?